src/DataProtection/DataProtection/test/Microsoft.AspNetCore.DataProtection.TrimmingTests/Constants.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+internal static class Constants
+{
+    public const string Password = "password";
```

Comment:
Credscan is going to flag anything that looks like a password/token/private key. I think it's fine to check in am suppress later, but fyi "placeholder" will not trigger it, while "password" will...

Reply:
These were copied from

https://github.com/dotnet/runtime/blob/4679e59ac4431380ff2455b471c9b3e9fda503e6/src/libraries/System.Net.Http.WinHttpHandler/tests/UnitTests/ClientCertificateHelper.cs#L176-L223

That code has a commented out SuppressMessage - not sure if that is doing anything.

If we don't use "password", we will need a new certificate.

Reply:
eh, fine for now

Reply:
I added a suppression for the cert, like in the above link. Not sure if I need one for the password. How often does Credscan run on our code?